cime_config/buildnml

```diff
@@ -1578,7 +1582,11 @@ cat >>! $RUNDIR/ocn_in$inststr << EOF
 !   EXPOCA         - Ca export production (epcalc100) [mol Ca m-2 s-1]
 !   EXPOSI         - Si export production (epsi100) [mol Si m-2 s-1]
 !   PCO2           - Surface PCO2 (spco2) [uatm]
-!   KWCO2          - kwco2 x solubility
+!   PCO2M          - Surface PCO2 under moist air assumption [uatm]
+!   KWCO2          - kwco2 
+!   KWCO2KH        - kwco2 x solubility (moist air)
+!   CO2KHD         - khd CO2 solubility under dry air assumption 
+!   CO2KH          - kh CO2 solubility under moist air assumption 
 !   CO2FXD         - Downward CO2 flux (co2fxd) [kg C m-2 s-1]
```

Comment:
Regarding the naming of of these variables, I find it confusing that sometimes a "D" for dry is appended, and sometimes a "M" for moist, and sometimes nothing. I would suggest to use only "M" to indicate the moist air assumption:

!   PCO2M            - Surface PCO2 under moist air assumption [uatm]
!   KWCO2            - kwco2 
!   KWCO2KHM    - kwco2 x solubility (moist air)
!   CO2KH             - khd CO2 solubility under dry air assumption 
!   CO2KHM          - kh CO2 solubility under moist air assumption 

I see that the idea was to use names that are close to the values in the code, but in the namelist, it should make sense for users that have no insights in the code. To resolve this one could also rename kh -> khm and khd -> kh in carchm, but not sure if this would make sense

Reply:
I can do this. One question though was/is the issue with previous names and potential confusion about it during subsequent cmorization steps. Any opinion/insights/knowledge about this? I tried to keep the former names, if possible.

Reply:
What is important is that PCO2 is not changed (since this was/is of course used in cmorization). Everything else is not CMIP6 output. So the only potential confusion is that KWCO2 will change its meaning, but this was actually very confusing before, so this is an improvement (not backwards-compatible though).

Reply:
I think this should still be fine, but we should remember to mention the change in KWCO2 in the CHANGELOG (for both BLOM/iHAMOCC-1.3.0 and NorESM-2.0.6)